pkg/transport/cmd/options.go

```diff
@@ -41,7 +44,8 @@ func NewTransportOptions() *TransportOptions {
 		Concurrency:            defaultConcurrency,
 		WdsClientOptions:       clientopts.NewClientOptions[*pflag.FlagSet]("wds", "accessing the WDS"),
 		TransportClientOptions: clientopts.NewClientOptions[*pflag.FlagSet]("transport", "accessing the ITS"),
-		MaxSizeWrappedObject:   500 * 1024,
+		MaxNumWrapped:          math.MaxInt,
```

Comment:
FYI, since there is a known upper bound on Kubernetes API object size, you could use that here (surely each wrapped object will occupy at least one byte) instead of importing `math`.